### PR TITLE
create bouncing dots tokens

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**
This makes available a first draft of the tokens needed to create the _bouncing dots_ loader mixin.

Story: https://phabricator.wikimedia.org/T267010

### CODE REVIEW

**Review by tzhelyazkova — COMMENTED** [review]
Hey @SaiSan-WMDE :wave: 
This looks good and could be merged. I understand it's a first draft, but we can safely merge it and introduce changes to it later if needed.

File: tokens/properties/mixins/BouncingDots.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            "duration": {
+                "value": "{animation.loader.duration.medium.value}"
+            },
+            "delay": {
+                "default": {
+                    "value": "{animation.loader.delay.medium.value}"
+                },
+                "before": {
+                    "value": "{animation.loader.delay.slow.value}",
+                    "comment": "or slow?"
```
Comment (SaiSan-WMDE) [inline tokens/properties/mixins/BouncingDots.json:39]:
```suggestion
```

File: tokens/properties/mixins/BouncingDots.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                },
+                "after": {
+                    "value": "{animation.loader.delay.none.value}",
+                    "comment": "or none?"
```
Comment (SaiSan-WMDE) [inline tokens/properties/mixins/BouncingDots.json:42]:
```suggestion
```

**Comment by SaiSan-WMDE** [comment]
Ok that sounds good, @tzhelyazkova. I don't think the loader mixin will be implemented any time soon, but this PR is linked to the ticket and I agree that changes could always be introduced later. Thanks for taking a look!

File: tokens/properties/mixins/BouncingDots.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            },
+            "delay": {
+                "default": {
+                    "value": "{animation.loader.delay.medium.value}"
+                },
+                "before": {
+                    "value": "{animation.loader.delay.slow.value}",
+                },
+                "after": {
+                    "value": "{animation.loader.delay.none.value}",
```
Comment (SaiSan-WMDE) [inline tokens/properties/mixins/BouncingDots.json:41]:
```suggestion
                    "value": "{animation.loader.delay.none.value}"
```

File: tokens/properties/mixins/BouncingDots.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        "animation": {
+            "duration": {
+                "value": "{animation.loader.duration.medium.value}"
+            },
+            "delay": {
+                "default": {
+                    "value": "{animation.loader.delay.medium.value}"
+                },
+                "before": {
+                    "value": "{animation.loader.delay.slow.value}",
```
Comment (SaiSan-WMDE) [inline tokens/properties/mixins/BouncingDots.json:38]:
```suggestion
                    "value": "{animation.loader.delay.slow.value}"
```

**Review by tzhelyazkova — APPROVED** [review]
Bootstrapping locally causes no errors so this is good to go! :rocket:

**Pull request merged**